### DISCUSSION TITLE
Would it be possible to provide a roadmap of current/planned supported Python runtimes?

### DISCUSSION THREAD

**Discussion by deric4** [aws-powertools/powertools-lambda-python#400] [Q&A] [answered]
As someone who's currently investing time in adopting this library, would it be possible to share with users  a roadmap of what to expect down the road in response to: https://github.com/samuelcolvin/pydantic/issues/2678 ?

Obviously Python 3.10 isn't a currently supported lambda runtime  and prob won't have any immediate impact, but having spent a fair bit of time evaluating, testing, and adopting Pydantic and/or aws-lambda-powertools across various projects, it'd be nice to have a blurb somewhere in the docs to check up on from time to time.

**Comment by mwarkentin** [comment]
👍 I think even outside of the specific pydantic issue, it would be nice to have a clear guide for which versions are planned to be supported. Will the library support all supported python lambda runtimes? Or will it be a subset over time?

**Reply by heitorlessa** [reply]
Hey everyone!

As per tenets, we follow officially supported runtimes by Lambda. We'll support 3.10 once Lambda does, and similarly will deprecate when others are EOL too.

As for roadmap, I'll be publishing an experiment on a Board in the next two to three weeks :)

**Comment by heitorlessa** [comment]
Update - today's release took additional cycles to get out. Now that's finally out there, we'll create a public project with some cards to share what we're thinking, what's under consideration, and what we're currently working on

**Comment by heitorlessa** [comment, marked as answer]
Here: https://github.com/awslabs/aws-lambda-powertools-roadmap

We'll update it this week as we migrate feature requests, RFCs, etc. but at least we have a more permanent place now for all runtimes ;)
